src/main/java/duke/common/Const.java

```diff
@@ -0,0 +1,13 @@
+package duke.common;
+
+import java.time.format.DateTimeFormatter;
+
+public final class Const {
```

Comment:
I think it would be better to use the full term for brevity?
```suggestion
public final class Constant {
```